examples/webserver/ratelimit/README.md

```diff
@@ -0,0 +1,66 @@
+# Helidon SE Rate Limiting Example
+
+This example demonstrates two mechanisms to perform request rate limiting in a Helidon SE application.
+The mechanisms shown here will also work with Helidon MP.
+
+The example does the following:
+
+1. Sets WebServer's `max-concurrent-requests` to 10
+2. Uses a `java.util.concurrent.Semaphore` to limit the number of requests processed concurrently to 5
+
+The server has one endpoint: `sleep` that sleeps for the specified number of seconds. The
+sleep operation is protected by a `Semaphore` to allow no more than five requests to be processed concurrently.
+
+These values are configured in `application.yaml`
+
+## Build and run
+
+Build and start the server:
+```shell
+mvn package
+java -jar target/helidon-examples-webserver-ratelimit.jar
+```
+
+The application logs the limits it is using:
+```
+2024.08.15 14:34:27.694          Application rate limit is 5
+2024.08.15 14:34:27.694 WebServer maxConcurrentRequests is 10
+```
+
+## Exercise the application
+
+Send a burst of 15 concurrent requests to the server:
+```
+for i in $(seq 15); do \
+  curl --noproxy '*' -o /dev/null -s -w "%{http_code}\n" http://localhost:8080/sleep/3 & \
```

Comment:
Does this work instead ?
```shell
curl -s  \
  --noproxy '*' \
  -o /dev/null \
  --parallel \
  --parallel-immediate \
  -w "%{http_code}\n" \
  "http://localhost:8080/sleep/3?c=[1-15]"
```

Reply:
Yes. I could not find a way to avoid mutating the URL on every request with `--parallel`, but this is still pretty clean. I'll use it.